### PR TITLE
feat : mark left members

### PR DESCRIPTION
**Pull request by prachipathrayan**

### CODE REVIEW

File: services/user.py [hunk 1]
```diff
@@ -115,17 +115,17 @@ def get_password():
...
         "data": {
-            "attributes": {"discord_id": str(member.id)},
+            "attributes": {"discord_id": member_id},
             "type": "users",
         }
     }
     try:
         resp = await send_request(method_type="PUT", url=url, data=myobj)
-        infoLogger.info("User status successfully updated")
+        infoLogger.info("request for user status update is successfully sent")
```
Comment (vaibhav353) [inline services/user.py:128]: Print member_id also

**Review by vaibhav353 — APPROVED** [review]
LGTM

File: listMembers.py [hunk 2]
```diff
@@ -21,6 +21,26 @@ async def listExistingMembers():
                 await submit_user_details(member)
 
 
+async def markLeftMembers():
+    res = await getAllMembers()
+    all_database_members_set = set()
+    for i in res["data"]:
```
Comment (ShipraShalini) [inline listMembers.py:27]: Try using list comprehension more.
```Python
all_database_members_set = {int(user["attributes"]["discord_id"]) for user in res["data"]}
all_members_on_discord_set = {member.id for member in client.get_all_members() if not member.bot}
```

Comment (ShipraShalini) [reply listMembers.py:27]: If all you need is left_members_set then there's no need to use two loops. From what I understand from the code this is what you want:
```Python
left_members = []
for member in client.get_all_members():
    if not member.bot and member.id not in all_database_members_set:
       if member.id in left_members:
           continue
       left_members.append(member.id)
       resp = await update_user_status(member.id)
           if resp:
                infoLogger.info("User status successfully updated")
           else:
                errorLogger.error("Error while updating user status")
```

**Pull request closed without merging**